diff --git a/configure.ac b/configure.ac
index 67da852667d..76c320292ae 100644
--- a/configure.ac
+++ b/configure.ac
@@ -7509,7 +7509,7 @@ AC_DEFUN
      libs_nsgui="-framework AppKit"
      if test "$NS_IMPL_COCOA" = "yes"; then
         libs_nsgui="$libs_nsgui -framework IOKit -framework Carbon \
-                    -framework IOSurface -framework QuartzCore"
+                    -framework IOSurface -framework QuartzCore -framework Speech"
         if test "$emacs_cv_macosx_12_0" = "yes"; then
 	  libs_nsgui="$libs_nsgui -framework UniformTypeIdentifiers"
 	fi
